plugins/modules/monitoring/uptimerobot.py

```diff
@@ -7,139 +7,183 @@
 __metaclass__ = type
 
 
-DOCUMENTATION = '''
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
 
+DOCUMENTATION = '''
 module: uptimerobot
-short_description: Pause and start Uptime Robot monitoring
+short_description: Create, remove, pause and start monitors in Uptimerobot
 description:
-    - This module will let you start and pause Uptime Robot Monitoring
-author: "Nate Kingsley (@nate-kingsley)"
+    - This module lets you create, remove, pause and start monitors in Uptimerobot
+author:
+- "Nate Kingsley (@nate-kingsley)"
+- "Ninjiner (@Ninjiner)"
+version_added: "1.9"
 requirements:
     - Valid Uptime Robot API Key
 options:
     state:
         description:
             - Define whether or not the monitor should be running or paused.
         required: true
-        choices: [ "started", "paused" ]
+        choices: [ "started", "paused", "created", "absent", "present" ]
+    url:
+        description:
+            - URL to be checked.
+        required: true
+        version_added: '2.10'
+    name:
+        description:
+            - The friendly name of the monitor.
+        required: true
+        version_added: '2.10'
+    check_type:
+        description:
+            - The kind of check, that will be performed on the url.
+        choices: [ "http", "ping" ]
+        version_added: '2.10'
     monitorid:
         description:
             - ID of the monitor to check.
-        required: true
+        required: false
     apikey:
         description:
             - Uptime Robot API key.
         required: true
 notes:
-    - Support for adding and removing monitors and alert contacts has not yet been implemented.
+    - Support for further functionalities, which the api provides has not yet been implemented.
+    - created and absent are added in version 2.10
```

Comment:
```suggestion
```
This should be mentioned in the `state` documentation.

Reply:
Changed in: https://github.com/ansible-collections/community.general/pull/961/commits/863c2e89453a20fcce643f8492160a4c4bba2b1d